packages/components/src/DateTimePickers/DateTime/Input/Input.component.js

```diff
@@ -10,19 +10,17 @@ export default function Input(props) {
 		inputManagement,
 		dateInputManagement,
 		timeInputManagement,
-		errorManagement,
 	} = useContext(DateTimeContext);
 	const partInputManagement = props.part === 'date' ? dateInputManagement : timeInputManagement;
-
+	const value = props.part === 'date' ? datetime.dateTextInput : datetime.timeTextInput;
 	return (
 		<DebounceInput
-			aria-describedby={errorManagement.inputErrorId}
+			size={partInputManagement.placeholder.length + 1}
```

Comment:
Here plus 1 to show the full placeholder text on input field. or placeholder overflow and ellipsis is shown